### ISSUE TITLE
Feature Suggestion votes recorded with Zero coins

### ISSUE THREAD

**Issue by cyberianfrontier** [worldanvil/worldanvil-bug-tracker#1298]
**Please make sure to fill out the template to the best of your ability. Add your information to the right of the colon as requested by the label! More information helps us track down the issue and fix it. Bug reports that do not let us reproduce the issue cannot be addressed. Thank you for your time in helping make World Anvil better!**

**World Anvil Username**: Buzzard

**Feature**: Feature Suggestions

**Operating System**: Windows 10

**Browser**: Chrome

**Related URLS** (To public & visible pages, add screenshots otherwise):
https://www.worldanvil.com/community/voting/suggestion/ce87ee84-6a43-419f-a12c-d00792a0f180/view
https://www.worldanvil.com/community/voting/suggestion/bf2d728d-64fc-4b59-8458-18fa8dfe3956/view

**Describe the Issue**
Two of my votes on Feature Suggestions recorded as Zero coins.  Both were cast as +1.  The feature I voted on before these two recorded normally.

**To Reproduce**
Seems to have started with my anonymous vote.

**Expected behavior**
Expected to be "charged" 1 coin each.

**Comment by iamromeo** [comment]
@cyberianfrontier have you selected a coin value and the result was 0 anyway?

**Comment by cyberianfrontier** [comment]
Yes.  In both instances, I chose the +1.  And I think that at least one of those submitted when I pressed the +1, instead of waiting for me to press the "vote" button.  I just had a look at the HTML and I suspect that this was a JS issue.  Perhaps an associated .js file didn't get loaded properly.  That's the only way I can think of that the vote form would get submitted by simply pressing one of the coin value <button>s.  And without JS to assign the number of coins, the vote recorded as zero.  So I guess this isn't a real bug.  Sorry to waste your time.
